### PR TITLE
Makes Admin AI interaction a logged toggle. Take 2

### PR DESCRIPTION
**Pull request by Shadowlight213**
Remake of #13534 
oranges did his whitespace thing and It's more effort to learn how to use command line git than it is to remake the PR.

### CODE REVIEW

File: code/_onclick/observer.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@
 // Oh by the way this didn't work with old click code which is why clicking shit didn't spam you
 /atom/proc/attack_ghost(mob/dead/observer/user)
 	if(user.client)
-		if(check_rights(R_ADMIN, 0))
+		if(check_rights(R_ADMIN, 0) && user.client.AI_Interact)
```
Comment (MrStonedOne) [inline code/_onclick/observer.dm]: why doesn't this use isadminghost()?

**Comment by tkdrg** [comment]
I'd like more extensive logs honestly, preferably everything the admin does should be logged. At the very least it would make sense to log which machine was used, since you can easily do that by calling a logging proc right before the `attack_ai()` inside `/atom/proc/attack_ghost()`.

**Comment by Shadowlight213** [comment]
@tkdrg I was thinking of that, but I didn't want to have it logging for stuff that isn't machines like floors and such.

**Comment by tkdrg** [comment]
True, this is at the /atom level. Well, this isn't the best, but some logging is better than no logging, and this is an useful feature for sure, so I'm merging it.

**Pull request merged**